js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties

```diff
@@ -192,3 +192,10 @@ recovery-codes-generate-new-codes=Generieren Sie neue Wiederherstellungscodes, u
 error-invalid-multivalued-size=Attribut {0} muss mindestens {1} und höchstens {2} Wert(e) haben.
 recovery-authn-code=Meine Authentifizierungscodes für die Wiederherstellung
 recovery-authn-codes-help-text=Diese Codes können verwendet werden, um Ihren Zugang wiederherzustellen, falls Ihre anderen Zwei-Faktor-Mittel nicht verfügbar sind.
+verifiableCredentialsIssuerAlert=War nicht in der Lage, die Ausstellungsinformationen abzurufen.
+verifiableCredentialsConfigAlert=War nicht in der Lage, die Berechtigungsnachweiskonfiguration abzurufen.
+oid4vci=Überprüfbare Berechtigungsnachweis
+verifiableCredentialsTitle=Überprüfbare Berechtigungsnachweis
```

Comment:
I think this should be at least a plural ("Berechtigungsnachweise"). 

@thomasdarimont - as one who is involved with Keycloak's OpenID Connect SIG and a German native speaker: Could you please comment how this should be translated, and if it should be translated at all?

Reply:
I agree with you @ahus1 that plural `Verifizierbare Berechtigungsnachweise` seems to be correct.
See: https://www.selfsovereignidentity.it/einfuhrung-in-verifiable-credentials/

Reply:
@ahus1 & @thomasdarimont  thanks for the clarification. Changes are incoming.

Reply:
Thank you for this translation!